### PR TITLE
Update home page text

### PR DESCRIPTION
**Pull request by amy-silcock**
@PetyaKangalova @jahnvidave I've updated the wording on the Registry homepage, as well as some of the links. (https://github.com/IATI/ckanext-iati/issues/170)

Please can you review and suggested edits.

@derwas no action needed for the moment. We will review this internally first.

Thank you, Amy

### CODE REVIEW

File: ckanext/iati/theme/templates/home/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     {{ self.flash() }}
 
     {% trans %}
-      <h1><strong>IATI Data</strong> is the place to search, register and use IATI data.</h1>
+      <h1><strong>IATI Registry</strong> is the place register your organisation and IATI datasets.</h1>
```
Comment (andylolz) [inline ckanext/iati/theme/templates/home/index.html]: > is the place register your organisation

looks like this is missing the word “to”

File: ckanext/iati/theme/templates/home/index.html [hunk 2]
```diff
@@ -38,8 +38,8 @@ <h2><a href="/using-iati-data">Using IATI data</a></h2>
         <a href="{{ h.url_for(controller='ckanext.iati.controllers.static:StaticController', action='dashboard') }}" class="block">Dashboard</a>
         <p>The IATI Dashboard provides statistics, charts and metrics on IATI data</p>
         <ul>
-          <li><a href="http://opendevtoolkit.org">See a list of projects, websites and tools that utilise IATI data</a></li>
-          <li><a href="/registry-api">Developer?  Resources to help.</a></li>
+          <li><a href="https://iatistandard.org/en/using-data/">Find out how use IATI data</a></li>
```
Comment (andylolz) [inline ckanext/iati/theme/templates/home/index.html]: > Find out how use IATI data

looks like this is missing the word “to”

**Comment by andylolz** [comment]
This fixes #170.

**Comment by andylolz** [comment]
This is great! I’ve just suggested a couple of small amendments.

File: ckanext/iati/theme/templates/home/index.html (hunk 1 above)
Comment (andylolz) [reply ckanext/iati/theme/templates/home/index.html]: In fact, I wonder if this should also say “**The** IATI Registry is the place **to** register […]”

Comment (derwas) [reply ckanext/iati/theme/templates/home/index.html]: @amy-silcock do you agree with @andylolz 's suggestion?

File: ckanext/iati/theme/templates/home/index.html (hunk 2 above)
Comment (derwas) [reply ckanext/iati/theme/templates/home/index.html]: @andylolz does this mean that the link to opendevtoolkit.org does not make sense anymore?

Please also note that the /registry-api is now redirected to http://iatistandard.org/en/using-data/IATI-tools-and-resources/using-IATI-registry/ (check this https://github.com/IATI/ckanext-iati/blob/ckan-2.8.1/ckanext/iati/plugins.py#L126)

Comment (andylolz) [reply ckanext/iati/theme/templates/home/index.html]: > @andylolz does this mean that the link to [opendevtoolkit.org](http://opendevtoolkit.org) does not make sense anymore?

@derwas Yes, that’s correct – [it was decommissioned](https://discuss.iatistandard.org/t/open-development-toolkit-is-being-decommissioned-on-18-sept/1444), and [now redirects](http://opendevtoolkit.org) to iatistandard.org.

**Comment by andylolz** [comment]
👍

**Comment by andylolz** [comment]
This PR was approved but never merged.

**Comment by amy-silcock** [comment]
@gtkChop please can this one be implemented on the Registry?

**Pull request merged**